include/SFML/Graphics/Texture.hpp

```diff
@@ -711,6 +711,14 @@ class SFML_GRAPHICS_API Texture : GlResource
     ////////////////////////////////////////////////////////////
     [[nodiscard]] static unsigned int getMaximumSize();
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the state of whether the Y orientatino of pixels are flipped
+    ///
+    /// \return `true` if pixels are flipped, otherwise `false`
+    ///
+    ////////////////////////////////////////////////////////////
+    [[nodiscard]] bool getPixelsFlipped() const;
```

Comment:
the name is confusing, when I read it I thought it returns a pointer to the pixels flipped but no this is just a boolean getter I think an isPixelsFlipped or something else is better name

Reply:
`isPixelsFlipped` or maybe `arePixelsFlipped`? I agree that a better name probably exists.

Reply:
How about:
- `m_isYFlipped`
- `bool isYFlipped()`

or something like that?